.github/workflows/build.yml

```diff
@@ -0,0 +1,133 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        pg_major: [ 12, 11, 10 ]
+        postgis_version: [ 3 ]
```

Comment:
We did... This is an artifact of some earlier builds that tested both PostGIS 2.5 and 3. Since we decided to stick with only testing the latest version, I'll move this back out to an environment variable.

Reply:
Great, thanks! If you could clean that up in efcore.pg as well that would be awesome.